### PR TITLE
Initial schema for YDBCP database

### PR DESCRIPTION
**Pull request by qrort**

### CODE REVIEW

File: internal/ydbcp-db-connector/read.go [hunk 1]
```diff
@@ -26,19 +25,20 @@ var (
 
 type YdbDriver interface {
 	SelectRunningBackups() chan types.Backup
```
Comment (robdrynkin) [inline internal/ydbcp-db-connector/read.go:27]: What for do we need a channel here? Why not slice?

File: internal/ydbcp-db-connector/read.go [hunk 2]
```diff
@@ -52,36 +52,38 @@ func InitTableClient(dsn string) table.Client {
...
+	err := d.ydbDriver.Table().Do(ctx, func(ctx context.Context, s table.Session) (err error) {
 		var (
 			res          result.Result
-			backup_id    uuid.UUID
-			operation_id uint64
+			backup_id    types.ObjectID
+			operation_id *string
 		)
 		_, res, err = s.Execute(
 			ctx,
```
Comment (robdrynkin) [inline internal/ydbcp-db-connector/read.go:77]: Why did you move some queries to .yql files, but not this query?

File: internal/ydbcp-db-connector/read.go [hunk 3]
```diff
@@ -26,19 +25,20 @@ var (
...
 	SelectRunningBackups() chan types.Backup
+	Close()
 }
 
 type YdbDriverImpl struct {
-	tableClient table.Client
+	ydbDriver *ydb.Driver
 }
 
 func MakeYdbDriver(config config.Config) *YdbDriverImpl {
```
Comment (robdrynkin) [inline internal/ydbcp-db-connector/read.go:35]: MakeYdbDriver -> NewYdbDriver

File: internal/ydbcp-db-connector/read.go [hunk 4]
```diff
@@ -52,36 +52,38 @@ func InitTableClient(dsn string) table.Client {
...
 			`
 			SELECT
-				backup_id,
+				id,
 		 		operation_id,
 			FROM
 				Backups
 			WHERE
-				status = "PENDING";
+				status = 'PENDING';
```
Comment (robdrynkin) [inline internal/ydbcp-db-connector/read.go:86]: Lets move such constants to the separate file
const STATUS_PENDING="PENDING"

File: internal/types/backup.go [hunk 5]
```diff
@@ -2,7 +2,30 @@ package types
 
 import "github.com/google/uuid"
 
+type ObjectID uuid.UUID
+
+func MustObjectIDFromBytes(b [16]byte) ObjectID {
```
[the remark this answers is not in the document]
Comment (qrort) [reply internal/types/backup.go:7]: YDB GO SDK returns `х[16]byte` when UUID is selected. We can explicitly parse it into `uuid.UUID`. This approach is taken from `kaczka` code: `ObjectID` is basically an interface for `uuid.UUID` with implicit convertions from `[16]byte`. https://stackoverflow.com/questions/12434565/built-in-helper-for-must-pattern-in-go hence the name

File: internal/ydbcp-db-connector/read.go (hunk 2 above)
Comment (qrort) [reply internal/ydbcp-db-connector/read.go:77]: .yql files need to be executed once, this is a different type of query. i'll probably put all of those in a separate .go file as constants later.

**Review by bma13 — APPROVED** [review]
